e2e/authentication/logout.spec.js

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
+// eslint-disable-next-line import/extensions
+import { resetDatabaseAndImportDataFromPath } from '../scripts/dts-import';
+
+test.describe('Log Out', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath({ filePath: './e2e/data/with-admin.tar' });
+    await page.goto('/admin');
+
+    await page.getByLabel('Email').fill('test@testing.com');
+    await page
+      .getByLabel('Password*', {
+        exact: true,
+      })
+      .fill('myTestPassw0rd');
+
+    await page.getByRole('button', { name: 'Login' }).click();
```

Comment:
I think we could make this a global utility, because almost every test will need login before. What do you think?